modules/articles/client/services/models/article.client.model.js

```diff
@@ -0,0 +1,19 @@
+(function () {
```

Comment:
@rhutchison What's the purpose of this file? 

We haven't decided on how we're going to move to a client-side model implementation. This isn't needed, since it's just a duplication of *article.client.service*